columnize_test.go

```diff
@@ -337,6 +338,47 @@ func TestGetStringFormat02(t *testing.T) {
 	}
 }
 
+func TestTextAlignment(t *testing.T) {
```

Comment:
I think it makes sense to write table driven tests as defined in https://github.com/giantswarm/fmt/blob/master/go/unit_tests.md.

Reply:
Since all but ~4 of these tests where provided by the upstream repo we forked from, I'd rather keep it simple.